DigitalLearningSolutions.Data/Enums/EnrolmentMethod.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DigitalLearningSolutions.Data.Enums
+{
+    public enum EnrolmentMethod
+    {
+        Self = 1,
```

Comment:
please add other values as well in the enum. Right now, how is it taking value for EnrolmentMethod.AdminOrSupervisor in the above code?